doc/application/application.rst

```diff
@@ -675,15 +675,48 @@ following procedure:
 #. Rebuild the application normally following the steps specified
    in `Build an Application`_ above.
 
-
+.. _application_run:
 
 Run an Application
 ******************
 
-An application image can be run on real or emulated hardware. The kernel has
-built-in emulator support for QEMU. It allows you to run and test an application
-virtually, before (or in lieu of) loading and running it on actual target
-hardware.
+An application image can be run on real or emulated hardware.
+
+Most boards supported by Zephyr let you flash a compiled binary using
+the ``flash`` target to copy the binary to the board and run it.
+Follow these instructions to flash and run an application on real
+hardware:
+
+#. Open a terminal console and navigate to the application directory
+   :file:`~/app`.
+
+#. Enter the following command to build and run the application on
+   your board, which you usually must attach via USB.
```

Comment:
Either "usually" or "must", but not "usually must".  I'd pick "usually"

Reply:
Whoops, nice catch; thanks.